modules/ROOT/pages/api-proxy-landing-page.adoc

```diff
@@ -4,123 +4,68 @@ include::_attributes.adoc[]
 endif::[]
 :keywords: proxy, api, auto-generated, raml, wsdl, apikit, http
 
-API proxies, also known as "autogenerated proxies", are meant to enhance the usability of API Manager in your Mule application.
+API Proxies function as intermediaries between the external applications and the backend server. When you deploy an API proxy in front of your API, the proxy adopts API gateway capabilities to secure the APIs using different types of xref:policies-landing-page.adoc[policies]. 
```

Comment:
```suggestion
API proxies function as intermediaries between the external applications and the backend server. When you deploy an API proxy in front of your API, the proxy adopts API gateway capabilities to secure the APIs using different types of xref:policies-landing-page.adoc[policies]. 
```

Reply:
Can you clarify what this means: "deploy an API proxy in front of your API"? Do you mean that you deploy the proxy before you deploy the API, or that the proxy is prepended to the API, or...?

Is the point simply that the proxy uses policies to protect the backend by acting as a gateway between it and the API?